services/nexus/nexus-version.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
 
 function isSnapshotVersion(version) {
-  const pattern = /(\d+\.)*\d-SNAPSHOT/
+  const pattern = /(\d+\.)*[0-9a-fA-F]-SNAPSHOT/
```

Comment:
Can the hash really include uppercase letters, i.e. can `A-F` be dropped from the regex?

Reply:
I've never seen it upper case for my usecases (with `sbt-dynver`).  I'll revert that.